draft-ietf-quic-transport.md

```diff
@@ -4331,11 +4331,10 @@ Each endpoint unilaterally decides if the spin bit is enabled or disabled for a
 connection. Implementations MUST allow administrators of clients and servers
 to disable the spin bit either globally or on a per-connection basis. Even when
 the spin bit is not disabled by the administrator, implementations MUST disable
-the spin bit for a given connection with a certain likelihood. The random
-selection process SHOULD be designed such that on average the spin bit is
-disabled for at least one eighth of network paths. The selection process
-performed at the beginning of the connection SHOULD be applied for all paths
-used by the connection.
+the spin bit for at least a sixteenth of connections with an expectation that
+the spin bit is disabled for at least one eighth of network paths. The selection
```

Comment:
> endpoints MUST disable their use of the spin bit on at least one in every 16 network paths, or for one in every 16 connection IDs. This ensures that the spin bit signal is disabled on approximately one in eight network paths.

Reply:
@martinthomson If we are to "ensure" that the spin bit signal would be disabled for approx. 1/8 of network paths, I think we need to require endpoints to select one in every 16 paths *at random*. Otherwise, the paths that would have the spin bit disabled could go below 1/8. For example, when there is only one client and one server talking to each other, and if both of them disable spin bit for every 16th connection they handle, then the spin bit would be enabled for only 1/16 of all the connections.

Reply:
Quite right.  

> endpoints MUST disable their use of the spin bit for a random selection of at least one in every 16 network paths, or for one in every 16 connection IDs. This ensures that the spin bit signal is disabled on approximately one in eight network paths.